app/src/main/java/com/p2p/wallet/auth/ui/username/ReservingUsernameFragment.kt

```diff
@@ -45,6 +48,22 @@ class ReservingUsernameFragment :
             youCanSkipTextView.text = buildClickableText()
             youCanSkipTextView.movementMethod = LinkMovementMethod.getInstance()
             youCanSkipTextView.highlightColor = Color.TRANSPARENT
+
+            usernameEditText.addTextChangedListener(object : TextWatcher {
```

Comment:
`usernameEditText.doAfterTextChanged { 
         presenter.checkUsername(it.toString())
 }
`